server/utils/auth/stateless_auth.py

```
@@ -159,7 +159,7 @@ def get_org_id_from_request() -> Optional[str]:
                         g._org_id_resolved = row[0]
                         return row[0]
         except Exception as e:
-            logger.warning(f"Error looking up org_id for user {user_id}: {e}")
+            logger.warning(f"Error looking up org_id for user {sanitize(user_id)}: {e}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)